### PR TITLE
Fix IConfiguration adapter

### PR DESCRIPTION
**Pull request by Arkatufus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -392,6 +410,96 @@ var host = new HostBuilder()
     });
 ```
 
+### Advanced Usage
```
Comment (Aaronontheweb) [inline README.md]: Rename to "special characters and case sensitivity"

File: src/Akka.Hosting/Configuration/ConfigurationHoconAdapter.cs [hunk 2]
```diff
@@ -15,20 +15,20 @@ namespace Akka.Hosting.Configuration
 {
     public static class ConfigurationHoconAdapter
     {
-        public static Config ToHocon(this IConfiguration config)
+        public static Config ToHocon(this IConfiguration config, bool normalizeKeys = true)
```
Comment (Aaronontheweb) [inline src/Akka.Hosting/Configuration/ConfigurationHoconAdapter.cs:18]: LGTM

**Review by Aaronontheweb — CHANGES_REQUESTED** [review]
We need to add a spec to test this change (handling of case sensitivity)

**Comment by Arkatufus** [comment]
@Aaronontheweb the spec for case sensitivity is there already

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Pull request merged**